include/boost/json/impl/object.ipp

```diff
@@ -23,6 +23,54 @@
 #include <type_traits>
 
 BOOST_JSON_NS_BEGIN
+namespace detail {
+
+template<class CharRange>
+std::pair<key_value_pair*, std::size_t>
+find_in_object(
+    object const& obj,
+    CharRange key) noexcept
+{
+    BOOST_ASSERT(obj.t_->capacity > 0);
+    if(obj.t_->is_small())
+    {
+        auto it = &(*obj.t_)[0];
+        auto const last =
+            &(*obj.t_)[obj.t_->size];
+        for(;it != last; ++it)
+            if( key == it->key() )
+                return { it, 0 };
+        return { nullptr, 0 };
+    }
+    std::pair<
+        key_value_pair*,
+        std::size_t> result;
+    BOOST_ASSERT(obj.t_->salt != 0);
+    result.second = detail::digest(key.begin(), key.end(), obj.t_->salt);
+    auto i = obj.t_->bucket(
+        result.second);
+    while(i != object::null_index_)
+    {
+        auto& v = (*obj.t_)[i];
+        if( key == v.key() )
+        {
+            result.first = &v;
+            return result;
+        }
+        i = access::next(v);
+    }
+    result.first = nullptr;
+    return result;
+}
+
+
+template
+std::pair<key_value_pair*, std::size_t>
+find_in_object<string_view>(
+    object const& obj,
+    string_view key) noexcept;
```

Comment:
What is this

Reply:
Explicit template instantiation. This puts the template instantiation into the compiled library.